apps/frontend/src/components/global/ExportHTMLModal.vue

```diff
@@ -231,12 +398,60 @@ export default class ExportHTMLModal extends Vue {
     });
   }
 
+  // Takes all available existing file data to use as default settings/data for outputData object
   resetOutputData() {
+    // Total control count
+    const controlCnt =
+      StatusCountModule.countOf(this.filter, 'Passed') +
+      StatusCountModule.countOf(this.filter, 'Failed') +
+      StatusCountModule.countOf(this.filter, 'Not Applicable') +
+      StatusCountModule.countOf(this.filter, 'Not Reviewed') +
+      StatusCountModule.countOf(this.filter, 'Profile Error');
+
+    // Calculate & set compliance level from control statuses
+    const MAX_DECIMAL_PRECISION = 2;
+    let compliance = 0;
+    // If controls exist, calculate compliance level
+    if (controlCnt > 0) {
+      // Formula: compliance = Passed/(Passed + Failed + Not Reviewed + Profile Error) * 100
+      // Truncate to hundredths decimal place
+      const calculatedCompliance =
```

Comment:
i'm already calculating this in heimdall2/apps/frontend/src/components/cards/compliancechart.vue - in order to ensure that that value is the same, maybe put the math in a utility function that can be called from both places or maybe even just have it be its own thing in the vuex store somewhere.

Reply:
Changed as requested.